tools/StaticAnalysis/StaticAnalysis.Netcore.csproj

```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="$(MSBuildThisFileDirectory)..\..\Repo.props" />
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Please investigate whether net6 and net6.0 are the same and make it consistent.

Reply:
According to https://learn.microsoft.com/en-us/dotnet/standard/frameworks#:~:text=6-,net6.0,-2.1 `net6.0` is the formal term.

Reply:
Nice catch, thank you!